### DISCUSSION TITLE
Integration of chatgpt?

### DISCUSSION THREAD

**Discussion by qkimchi** [chatwoot/chatwoot#6809] [Feature Requests]
Hi,

Is integration of chatgpt in the pipeline?

Can it be integrated easily?

Could potentially be quite useful as a front line chatbot and service agent to then be followed by live human if still needed.

**Comment by qkimchi** [comment]
No response or feedback?

**Comment by qkimchi** [comment]
Anyone?

**Comment by sojan-official** [comment]
Please follow the updates here at https://github.com/chatwoot/chatwoot/issues/6436
